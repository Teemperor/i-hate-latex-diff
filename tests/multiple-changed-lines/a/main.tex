
\documentclass[12pt]{article}
\begin{document}
Hello world!
Goodbye moon!
\end{document}
