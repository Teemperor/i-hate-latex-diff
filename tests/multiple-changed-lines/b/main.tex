
\documentclass[12pt]{article}
\begin{document}
Hello world 2!
Goodbye moon 2!
\end{document}
