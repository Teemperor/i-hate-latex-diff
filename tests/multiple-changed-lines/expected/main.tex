
\documentclass[12pt]{article}
\begin{document}
\removedChange{Hello world!}
\addedChange{Hello world 2!}
\end{document}
